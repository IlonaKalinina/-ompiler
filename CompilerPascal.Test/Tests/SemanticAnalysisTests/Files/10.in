program Test_10;
var
a : string;
begin
a := 'one'/'two';
end.