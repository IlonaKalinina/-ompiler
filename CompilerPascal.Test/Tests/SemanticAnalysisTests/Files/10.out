(5,17) ERROR: Operator is not overloaded
